packages/tools/fluid-runner/tsconfig.bin.json

```diff
@@ -0,0 +1,9 @@
+{
+	"extends": "../../../common/build/build-common/tsconfig.node16.json",
+	"include": ["bin/*"],
+	"references": [
```

Comment:
why is tsconfig.bin.json required?

Reply:
We need some TS project file to provide coverage with eslint. Probably should rename it to have .`lint` in the name like was done for CJS test case.